electrum/gui/qml/qeapp.py

```diff
@@ -210,6 +210,10 @@ def doShare(self, data, title):
         it = jIntent.createChooser(sendIntent, cast('java.lang.CharSequence', jString(title)))
         jpythonActivity.startActivity(it)
 
+    @pyqtSlot()
```

Comment:
missing result type for the decorator, resulting in setMaxScreenBrightness() never actually getting called.

fixed in https://github.com/spesmilo/electrum/commit/7fa204f16b08a3c58ec403d9e07661531b91b773